test/testsid.pem

```diff
@@ -34,5 +34,5 @@ d/xgQ6J+/Z2kAgQAqQQCAgEsqoHDBIHAPGxCacLRMYb3hbL3lRqQIfH3xoufIwKn
 zOAIyjxH7GPZpI87/e93AF1uw6eyy+aWJM6G/71E8Ln1iQtZjrGkGQCjEcIrUhEk
 Bgg93A28zly2zu2aBwi2yT4bUcyHxE3P3Q4R8xkQxrdWSMfCjCy9HKUViOENKET6
 avKXF7lDT1iG/x8RtTHtFGYX4YcFVhdEqcqnZ7tw5sqkHG2D0gLzO2SETF9KPHCq
-zAHQtw5r4Dgl+a+BGLczh6as4yIH+xH+swMCAQA=
+zAHQtw5r4Dgl+a+BGLczh6as4yIH+xH+vAMCAQA=
```

Comment:
there are still failures in test_sid. you may need to regenerate the test sid because of failures seen 25-test_sid
the command you want to use is:
```
LD_LIBRARY_PATH=`pwd` ./apps/openssl sess_id -in test/testsid.pem -out testsid.pem
mv testsid.pem test/testsid.pem
```